pt_miniscreen/core/components/list.py

```diff
@@ -60,6 +60,11 @@ def __init__(
         end_index = self.state["num_visible_rows"] + self.state["top_row_index"]
         self.rows = [self.create_child(Row) for Row in Rows[start_index:end_index]]
 
+    @property
+    def components_to_top_row(self):
```

Comment:
This is used to figure out the scroll distance needed to scroll to the bottom right? Maybe something like "distance_to_bottom" and "distance_to_top" would be a bit friendlier. That terminology could also be used in "scroll_to_bottom" and "scroll_to_top" methods.